src/qaqc/download_and_flag_fxns/add_biofilm_flag.R

```diff
@@ -1,13 +1,20 @@
 add_biofilm_flag <- function(df){
 
-  # We only look for drift due to biofilm growth on turbidity sensors...
-
-  if(unique(df$parameter == "Turbidity")){
-
-    # Define a function to check if a given 3-day window's worth of data has a linear
-    # relationship with time
+  # Only test for biofilm growth on turbidity sensors
+  if("Turbidity" %in% df$parameter){
+
+    # subset data to turbidity and conductivity only
+    sub <- df %>%
+      dplyr::filter(parameter %in% c("Turbidity", "Specific Conductivity")) %>%
+      dplyr::select(DT_round, DT_join, parameter, mean) %>%
+      tidyr::pivot_wider(names_from = parameter, values_from = mean)
```

Comment:
Question: at what point does this function get passed into the pipeline? As I understand the pipeline the dfs go into the flag_all_data() step in site_param form. Is there a step that binds these parameters together that I missed?

Reply:
This happens AFTER the flag_all_data step. See my 00_preparatory_workflow to see where I'm envisioning these interasensor checks happening